templates/Gemfile

```diff
@@ -9,4 +9,5 @@ end
 group :development do
   # Access an IRB console on exception pages or by using <%= console %> in views
   gem "web-console", "~> 2.0"
+  gem 'listen', '~> 3.1', '>= 3.1.5'
```

Comment:
I don't know anything about listen, is it needed by default, or just helpful to have?

Reply:
Needed by default. I had to add it to get `bundle install` to run.

Reply:
Very odd, anyway, probably not worth looking in to.